netlify.toml

```diff
@@ -3,6 +3,14 @@ from = "/*"
 to = "/"
 status = 200
 
+# we use content based hash for filenames
+# 31536000 is 1 year, recommended by cra
+# cra doc https://create-react-app.dev/docs/production-build/
+[[headers]]
+  for = "/static/*"
+  [headers.values]
+    Cache-Control = "max-age=31536000"
```

Comment:
This looks like a performance optimisation to not have to redownload assets that haven't changed, and NOT like sth related to *missing* chunk files. Or am I missing something here?